package/windows/CMakeLists.txt

```diff
@@ -1,7 +1,9 @@
 project(package_windows)
 
 if(WIN32)
-    add_executable(launchdf WIN32 launchdf.c)
-    install(TARGETS launchdf
-        DESTINATION ${DFHACK_DATA_DESTINATION})
+    include_directories(${STEAMAPI_SOURCE_DIR})
+    link_libraries(${STEAMAPI_LIBRARY})
+    add_executable(launchdf WIN32 launchdf.cpp)
+    install(TARGETS launchdf DESTINATION ${DFHACK_DATA_DESTINATION})
+    install(FILES ${STEAMAPI_SHARED_LIBRARY} DESTINATION ${DFHACK_DATA_DESTINATION})
```

Comment:
Is this a good idea? this is the same situation we're in now with SDL.dll. Uninstalling DFHack will remove the steam dll and render DF unstartable. Do we have to install it? Won't DF already have a copy we can use?

Reply:
`${DFHACK_DATA_DESTINATION}` is is the `hack` folder

Reply:
ok, good.